lib/job-runner/src/main.rs

```diff
@@ -28,6 +28,7 @@ fn main() -> anyhow::Result<()> {
 	let root_user_enabled = std::env::var("NOMAD_META_root_user_enabled")
 		.context("NOMAD_META_root_user_enabled")?
 		== "1";
+	let runner = std::env::var("NOMAD_META_runner").unwrap_or("job_run".to_string());
```

Comment:
This should work for any existing usage of job-runner, while the new dynamic servers passes in a new var.